src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -248,8 +248,7 @@ private void InitializeOpenBuffers(SnapshotSpan triggerSpan)
             this.UndoManager.CreateInitialState(this.ReplacementText, _triggerView.Selection, new SnapshotSpan(triggerSpan.Snapshot, startingSpan));
             _openTextBuffers[triggerSpan.Snapshot.TextBuffer].SetReferenceSpans(SpecializedCollections.SingletonEnumerable(startingSpan.ToTextSpan()));
 
-            UpdateReferenceLocationsTask(_threadingContext.JoinableTaskFactory.RunAsync(
-                () => _renameInfo.FindRenameLocationsAsync(_options, _cancellationTokenSource.Token)));
+            UpdateReferenceLocationsTask();
```

Comment:
this pattern also follows what we do with our other task kick-off code.  No need to pass tasks to tasks.  We simply ask for the work to be setup on the UI threads, and the helper sets things up correctly based on the state of this object.